queries.sql

```diff
@@ -1,3 +1,46 @@
 /*Queries that provide answers to the questions from all projects.*/
 
-SELECT * from animals WHERE name = 'Luna';
+-- Select animals with names ending in 'mon'
+SELECT * FROM animals WHERE name LIKE '%mon';
+
+-- Select names of animals born between 2016 and 2019
+SELECT name FROM animals WHERE date_of_birth BETWEEN '2016-01-01' AND '2019-12-31';
+
+-- Select names of neutered animals with less than 3 escape attempts
+SELECT name FROM animals WHERE neutered = TRUE AND escape_attempts < 3;
+
+-- Select the date of birth for animals with names 'Agumon' and 'Pikachu'
+SELECT date_of_birth FROM animals WHERE name IN ('Agumon', 'Pikachu');
+
+-- Select names and escape attempts of animals with weight over 10.5 kg
+SELECT name, escape_attempts FROM animals WHERE weight_kg > 10.5;
+
+-- Select all neutered animals
+SELECT * FROM animals WHERE neutered = TRUE;
+
+-- Select all animals except 'Gabumon'
+SELECT * FROM animals WHERE name NOT IN ('Gabumon');
+
+-- Select animals with weight between 10.4 kg and 17.3 kg
+SELECT * FROM animals WHERE weight_kg BETWEEN 10.4 AND 17.3;
+
```

Comment:
- [ ] @DevKaranJ Kindly add queries for the section on the screenshot below, you have written half the queries, also consider updating the screenshots after fixing the errors when entering the data, this will ensure your screenshot results are correct and you have screenshots for all queries.
![image](https://github.com/DevKaranJ/vet-clinic/assets/61408860/360935cb-0693-4fe0-9ba8-1d522efdb721)